sdk/core/Azure.Core/samples/Response.md

```diff
@@ -127,6 +127,51 @@ await foreach (Page<SecretProperties> page in allSecretProperties.AsPages())
 }
 ```
 
+## Using System.Linq.Async with AsyncPageable
+
+The [`System.Linq.Async`](https://www.nuget.org/packages/System.Linq.Async) package provides a set of LINQ methods that operate on `IAsyncEnumerable<T>` type.
+Because `AsyncPageable<T>` implements `IAsyncEnumerable<T>` you can use `System.Linq.Async` to easily query and transform the data.
+
+### Convert to a `List<T>`
+
+`ToListAsync` can be used to convert an `AsyncPageable` to a `List<T>`.
```

Comment:
```suggestion
`ToListAsync` can be used to convert an `AsyncPageable` to a `List<T>`.  This might make several service calls if the data isn't returned in a single page.
```